todo_app/app.py

```diff
@@ -1,61 +1,193 @@
+    Boards = (json.loads(Boards.text))
+    for Board in Boards:
+        if Board['name'] == BoardName:
+            Board = Board['id']
+            break
+    else:
+        Board = None
+    return Board
+
+def find_list(trelloBoard,listName):
+    url = rootUrl+"/boards/"+trelloBoard+"/lists"
+    query = {
+        'key': Config.KEY,
+        'token': Config.TOKEN}
+    Boards = requests.request(
+        "GET",
+        url,
+        headers=headers,
+        params=query
+    )
+    Lists = (json.loads(Boards.text))
+    for List in Lists:
+        if List['name'] == listName:
+            List = List['id']
+            break
+    else:
+        List = None
+    return List
+
+def get_List(id):
+    url = rootUrl+"/lists/"+id
+    query = {
+        'key': Config.KEY,
+        'token': Config.TOKEN}
+    getList = requests.request(
+        "GET",
+        url,
+        headers=headers,
+        params=query
+    )
+    output = (json.loads(getList.text))
+    return output   
+
+def get_Items(todo_list):
+    url = rootUrl+"/lists/"+todo_list+"/cards"
+    query = {
+        'key': Config.KEY,
+        'token': Config.TOKEN}
+    cards = requests.request(
+        "GET",
+        url,
+        headers=headers,
+        params=query
+    )
+    output = (json.loads(cards.text))
+    return output
+
+def add_Item(todo_list,name,desc):
+    url = rootUrl+"/cards"
+    query = {
+        'key': Config.KEY,
+        'token': Config.TOKEN}
+    query['idList'] = todo_list
+    query['name'] = name
+    query['desc'] = desc
+    newCard = requests.request(
+        "POST",
+        url,
+        headers=headers,
+        params=query
+    )
+    output = (json.loads(newCard.text))
+    return output
+
+def get_Item(id):
+    url = rootUrl+"/cards/"+id
+    query = {
+        'key': Config.KEY,
+        'token': Config.TOKEN}
+    getCard = requests.request(
+        "GET",
+        url,
+        headers=headers,
+        params=query
+    )
+    output = (json.loads(getCard.text))
+    return output
+
+def set_Item(itemId,name,desc,state):
+    url = rootUrl+"/cards/"+itemId
+    query = {
+        'key': Config.KEY,
+        'token': Config.TOKEN}
+    query['name'] = name
+    query['desc'] = desc
+    query['idList'] = state
+    setCard = requests.request(
+        "PUT",
+        url,
+        headers=headers,
+        params=query
+    )
+    output = (json.loads(setCard.text))
+    return output
+
+trelloBoard = get_BoardId("SqmIler01")
+if not trelloBoard:
+    print("Board not found")
+    exit()
+todo_list = find_list(trelloBoard,"To Do")
+todoItems = get_Items(todo_list)
+doneItems = get_Items(find_list(trelloBoard,"Done"))
 
 @app.route('/',methods = ['POST', 'GET'])
 def index():
     if request.method == 'GET':
-        return render_template('index.html', items=get_items())
+        return render_template('index.html',
+            todoItems=get_Items(todo_list),
+            doneItems = get_Items(find_list(trelloBoard,"Done")))
```

Comment:
```suggestion
            todoItems=get_Items(find_list(trelloBoard,"To Do"))),
            doneItems = get_Items(find_list(trelloBoard,"Done")))
```
It's a bit weird for todoItems to rely on the todo_list variable defined elsewhere but doneItems fetches the list itself.

Best is for index() to fetch everything it needs to fetch, so you don't need to restart the server if something changes. Marginal hit to performance which you would solve with caching in reality.